src/main.cpp

```diff
@@ -62,12 +62,13 @@ int main(int argc, char* argv[]){
         InputFileWithTimeInterval<PTCL::RealPtcl>(sph_system, sysinfo);
         PROBLEM::setEoS(sph_system);
     }
+    PS::F64 output_interval = parameter_file.getValueOf("output_interval",100.);
```

Comment:
Does it make sense to change the default value to the value the output_interval was before? I.e. the default end time is 1e4, and the old number of snapshots was set to 200, so maybe make 50 the default output interval? Or is 100 preferable in some way?

Reply:
That's an excellent idea. I didn't think about calculating what it was before by default. I'll implement this.